cpp/ql/test/library-tests/ir/ir/PrintAST.expected

```diff
@@ -5122,19 +5122,25 @@ ir.cpp:
 #  809|                 Conversion = glvalue conversion
 #  809|                 Type = const Base
 #  809|                 ValueCategory = lvalue
-#  809|               expr: call to Base
-#  809|                   Type = void
-#  809|                   ValueCategory = prvalue
-#  809|                 0: (reference to)
-#  809|                     Type = const Base &
+#  809|               expr: (reference dereference)
```

Comment:
Is this diff the result of a bad merge/rebase? I don't see how your changes could cause it, and it looks similar to a diff from a recent EDG patch.

Reply:
I had an old copy of the extractor - I'll update it and rerun